packages/admin/src/RouterTabs.tsx

```diff
@@ -10,12 +10,15 @@ import { StackApiContext, StackBreadcrumb, StackSwitchApiContext } from "./stack
 
 interface ITabProps extends TabProps {
     path: string;
-    label: string;
+    label: React.ReactNode;
     forceRender?: boolean;
+    /**
+     * @deprecated Use label instead.
+     */
     tabLabel?: React.ReactNode;
```

Comment:
Should we add the deprecation in our changelog too ?

Reply:
And we could also log deprecation Notizen (off by default)

And create codemods